scripts/cluster/setup_master_node.sh

```diff
@@ -60,4 +60,17 @@ fi
 kubectl apply --filename $ROOT/configs/knative_yamls/serving-default-domain.yaml
 
 kubectl apply --filename https://github.com/knative/net-istio/releases/download/v0.19.0/release.yaml
-kubectl --namespace istio-system get service istio-ingressgateway
+
```

Comment:
we should have the same version for serving and eventing since Knative folks seem to have them in sync. please use one variable for both.

Reply:
done